draft-ietf-babel-information-model.md

```diff
@@ -331,23 +368,21 @@ babel-mcast-group:
   to expose this parameter as read-only ("ro").
 
 
-
 ## Definition of babel-interfaces-obj
 
 
 ~~~~
   object {
        string               ro babel-interface-reference;
       [boolean              rw babel-interface-enable;]
-       int                  rw babel-link-type;
+       string               rw babel-link-type;

```

Comment:
If your link types are well defined (enumerations), would this not be an enum.

Reply:
The information model (and BBF TR-181) don't have a specific "enum" type. Just strings that are enumerated.